docs/reference/filters/money/index.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Money
+parent: Filters
+---
+
+When using a `money` filter you have access to the following attributes.
+
+# money
+
+The provided integer returned as a formatted string including the registered currency.
+
+##### input
+{% raw %}
+```liquid
+  <h1>{{ 150 | money }}</h1>
+```
+{% endraw %}
+
+##### output
+Assuming the registered currency is GBP
+
+{% raw %}
+```html
+  <h1>£1.50</h1>
+```
+{% endraw %}
+
+# apply_promotion
+
+The discounted [Price]({% link docs/reference/objects/product/price.md %}) of a provided [Price]({% link docs/reference/objects/product/price.md %}) with the provided [Promotion]({% link docs/reference/objects/product/promotion.md %}) applied.
+
+##### input
+{% raw %}
+```liquid
+  {% assign reduced_price = price | apply_promotion: promotion %}
+  <h1>{{ reduced_price.fractional }}</h1>
+```
+{% endraw %}
+
+##### output
+Assuming an initial price of `120.00` and a percentage discount of `25%`
+
+{% raw %}
+```html
+  <h1>9000</h1>
+```
+{% endraw %}
+
+# extract_money_decimal
+
+The decimal integer of a given money string with currency code.
+
+##### input
+{% raw %}
+```liquid
+  <h1>{{ '$22,221,001.99' | extract_money_decimal }}</h1>
```

Comment:
It might be worth including a more realistic string here to indicate how it can be used, e.g. `<h1>{{ '$1,020.89/pp' | extract_money_decimal }}</h1>`